web-common/.factorypath

```diff
@@ -0,0 +1,243 @@
+<factorypath>
```

Comment:
Delete this file

Reply:
Don't know how this appeared here. But anyway I'll remove it